scipy/stats/tests/test_stats.py

```diff
@@ -7336,6 +7357,20 @@ def test_collapse(self):
         assert_almost_equal(
             stats.wasserstein_distance(u, v, u_weights, v_weights),
             np.average(np.abs(u), weights=u_weights))
+        # test collapse for n dimentional values
+        # Collapsing a n-D distribution to a point distribution at zero
+        # is equivalent to taking the average of the norm of data.
+        u = np.arange(-10, 30, 0.4).reshape((-1, 10))
```

Comment:
Is there something special about `arange` or can these be random?

Reply:
No I think I was simply following the existing example there, already updated that with random values.